.gitconfig

```diff
@@ -0,0 +1,47 @@
+[core]
```

Comment:
This file just adds some convenient stuff for `git`

For example run `git lg` to see a list of commit history...or `use `git cm` as a shorthand for `git commit -m`